### PR TITLE
Release 2.0.3

### PR DESCRIPTION
**Pull request by ryonakano**

### CODE REVIEW

File: com.github.ryonakano.pinit.yml [hunk 1]
```diff
@@ -7,13 +7,13 @@ finish-args:
   - '--share=ipc'
   - '--socket=wayland'
   - '--socket=fallback-x11'
-  - '--filesystem=~/.local/share/applications'
+  - '--filesystem=home'
```
Comment (bbhtt) [inline com.github.ryonakano.pinit.yml:10]: Why was this changed to home?

Comment (ryonakano) [reply com.github.ryonakano.pinit.yml:10]: Because we've faced several issues due to restricting access to `~/.local/share/applications`:

- Created desktop entries disappear after reboot/upgrade
    - https://github.com/ryonakano/pinit/issues/113
    - https://github.com/ryonakano/pinit/issues/64
- The path displayed in the text entry in the app is weird (/run/user/1000/...)
    - https://github.com/ryonakano/pinit/issues/61
        - I guess this is because the sandboxed app sees the exported path by the Document portal, but users expects the "correct" path to their executable/icon file is shown.

The app uses `~/.local/share/applications` to read/write desktop files stored inside that directory, but plus needs to know the path to the executable/icon files chosen by user. Before recently I thought the exported path by the portal is enough but found it's not from the above issue reports. So I changed the filesystem access to the entire HOME, where users often place their executable/icon files.

**Comment by razzeee** [comment]
> Before recently I thought the exported path by the portal is enough but found it's not from the above issue reports. 

Might be worth raising that over at https://github.com/flatpak/xdg-desktop-portal if you haven't already done that

File: com.github.ryonakano.pinit.yml (hunk 1 above)
Comment (razzeee) [reply com.github.ryonakano.pinit.yml:10]: > Before recently I thought the exported path by the portal is enough but found it's not from the above issue reports. 

Might be worth raising that over at https://github.com/flatpak/xdg-desktop-portal if you haven't already done that

Comment (bbhtt) [reply com.github.ryonakano.pinit.yml:10]: Getting the real path from portal is tracked by https://github.com/flatpak/xdg-desktop-portal/issues/475

The icon has no solution other than exposing common locations as they can be anywhere.

> needs to know the path to the executable

This can't be solved, flatpak does not have access to hosts `/usr/bin` `/bin` etc. at best you can get a `/run/host/usr` path.

> Pinned entries do not persist after reboot  ryonakano/pinit#113

This one is a bit weird, since once you save the desktop file it shouldn't be lost from `~/.local/share/applications`.

Comment (ryonakano) [reply com.github.ryonakano.pinit.yml:10]: > Getting the real path from portal is tracked by [flatpak/xdg-desktop-portal#475](https://github.com/flatpak/xdg-desktop-portal/issues/475) Exported paths won't survive restarts, only session.

Thank you for your info, I didn't know it's a known issue and also the exported paths are not permanent. I'll keep track of that issue.

> This can't be solved, flatpak does not have access to hosts `/usr/bin` `/bin` etc. at best you can get a `/run/host/usr` path or rely on executables being in home.

I meant `*.appimage` or `*.sh` files that user downloaded/created in their home directory, so just getting the real full paths to them is enough (read/write access to these paths are even not necessary). Accessing to `/usr/bin` or `/bin` is not required.

> This one is a bit weird, since once you save the desktop file it shouldn't be lost from `~/.local/share/applications`.

I guess this is because the paths to executable/icon files written in the desktop file become invalid after reboot (since these paths don't survive as you let me know). Maybe the application launcher regards these desktop files as broken and thus they won't be shown.

Comment (bbhtt) [reply com.github.ryonakano.pinit.yml:10]: Likely, I approved the change, as these are genuine constraints for a desktop file editor.
